tests/common-api.ts

```diff
@@ -34,7 +35,7 @@ export const getBalance = async (address: string, assetId?: string): Promise<big
     balance = BigInt(balanceObj.unwrap().balance.toBigInt());
   } else {
     const balanceObj = (await api.query.system.account(address)) as FrameSystemAccountInfo;
-    balance = BigInt(balanceObj.data.free.toBigInt() - balanceObj.data.feeFrozen.toBigInt());
+    balance = BigInt(balanceObj.data.free.toBigInt() - balanceObj.data.flags.toBigInt());
```

Comment:
I don't think the result is correct in this case since `flags` doesn't represent a Balance. See https://github.com/paritytech/substrate/blob/b7c18f1f5c9dc5c4ffa683e230303f084efe82cb/frame/balances/src/types.rs#L108

Reply:
Not just data structure was changed, also meaning of values have been changed. I believe you should use `frozen` property for this calculation.

Since Astar runtime is updated I will make some changes on the `AccountData` on the Portal today or tomorrow. The fix I made after Shibuya runtime upgrade was just temporary.

Reply:
@bobo-k2  CC: @gluneau 
Does this PR makes sense to you?
https://github.com/AstarNetwork/astar-apps/pull/910/files

Reply:
also, I've updated to use `frozen` in this branch
https://github.com/AstarNetwork/astar-apps/pull/909/commits/4e6ca4b11b848bc5f1c1e36ebc7f5fb5d01e1914